packages/fiori-elements-writer/package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "@sap-ux/fiori-elements-writer",
+    "version": "0.0.1",
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "rimraf dist && tsc",
```

Comment:
Should we include rimraf as a dependency?

Reply:
No, its no longer required.

Reply:
It's included at the root package.json.